packages/flet_local_auth/lib/src/local_auth.dart

```diff
@@ -0,0 +1,69 @@
+import 'dart:io' show Platform;
+
+import 'package:flet/flet.dart';
+import 'package:flutter/widgets.dart';
+import 'package:local_auth/local_auth.dart';
+
+class LocalAuthenticationControl extends StatefulWidget {
+  final Control? parent;
+  final Control control;
+  final FletControlBackend backend;
+
+  const LocalAuthenticationControl(
+      {super.key,
+      required this.parent,
+      required this.control,
+      required this.backend});
+
+  @override
+  State<LocalAuthenticationControl> createState() =>
+      _LocalAuthenticationControlState();
+}
+
+class _LocalAuthenticationControlState
+    extends State<LocalAuthenticationControl> {
+  @override
+  Widget build(BuildContext context) {
+    debugPrint("LocalAuthentication build: ${widget.control.id}");
+
+    () async {
+      final LocalAuthentication auth = LocalAuthentication();
+
+      widget.backend.subscribeMethods(widget.control.id,
+          (methodName, args) async {
+        switch (methodName) {
+          case "available":
+            // get's available biometrics
+            final List<BiometricType> availableBiometrics =
+                await auth.getAvailableBiometrics();
+            // check's if device is supported
+            final bool canAuthenticate = availableBiometrics.isNotEmpty;
+            debugPrint("canAuthenticate: $canAuthenticate");
+            return "$canAuthenticate";
+
+          case "authenticate":
+            try {
+              if (Platform.isWindows) {
+                args['biometricsOnly'] = "false";
+              }
+              final bool didAuthenticate = await auth.authenticate(
+                // if no title is given, uses "Authentication required" as the title
+                localizedReason: args['title'] ?? "Authentication required",
+                options: AuthenticationOptions(
```

Comment:
Can you expose all the other options of this class? Let's give the dev as much control over it as possible.

Reply:
the only other useful option is `sensitiveTransaction` (Whether platform specific precautions are enabled. For instance, on face unlock, Android opens a confirmation dialog after the face is recognized to make sure the user meant to unlock their device.
) I can expose that if it is required, but it is `true` by default so I don't see a reason do so.

Reply:
Let's try to expose all properties possible (especially simple ones like booleans, strings or numbers). The developer should be the one to choose :)

Reply:
ok.